src/course-unit/hooks.jsx

```diff
@@ -42,12 +44,20 @@ export const useCourseUnit = ({ courseId, blockId }) => {
     },
   };
 
+  useEffect(() => {
```

Comment:
Can you place both `useEffect`s near to each other?

Reply:
Transferred 👍